core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1402,7 +1417,7 @@ public function delete($blnDoNotRedirect=false)
 			$this->redirect($this->getReferer());
 		}
 
-		$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_DELETE, new DataContainerSubject($this->strTable, $this->intId));
+		$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new DeleteAction($this->strTable, $this->intId));
```

Comment:
second argument to `DeleteAction` must be array not string/int

Reply:
Fixed in 81aa45a7019e89b1cbc0e16fb6b3a8c44fc7b6cb.